packages/react/src/BacktraceClient.ts

```diff
@@ -1,24 +1,32 @@
-import { BacktraceConfiguration, BacktraceClient as BrowserClient } from '@backtrace/browser';
+import { BacktraceClient as BrowserClient, BacktraceConfiguration } from '@backtrace/browser';
 import { BacktraceReactClientBuilder } from './builder/BacktraceReactClientBuilder';
 
 export class BacktraceClient extends BrowserClient {
```

Comment:
I think the `private static _instance?: BacktraceClient;` accidentally got deleted

Reply:
oh, no - it's available via browser backtrace client.

Reply:
Ohhh, that's right. My bad!